rowhammer_tester/scripts/rowhammer.py

```diff
@@ -299,8 +332,15 @@ def main(row_hammer_cls):
         data_inversion=args.data_inversion,
     )
 
+    log_dir = Path(args.log_dir)
+    if not log_dir.is_dir():
+        log_dir.mkdir(parents=True)
+    row_hammer.log_directory = args.log_dir
```

Comment:
Just to double check, if no log-dir is provided, the `row_hammer.log_directory` will be `None`, correct?

Reply:
Yes, it will be `None` then. And in fact this should be checked before checking if the path exists, so I've just added one fix more.